extensions/grpc/deployment/src/test/java/io/quarkus/grpc/server/devmode/DevModeTestRestResource.java

```diff
@@ -0,0 +1,33 @@
+package io.quarkus.grpc.server.devmode;
+
+import javax.inject.Inject;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+/**
+ * @author Michal Szynkiewicz, michal.l.szynkiewicz@gmail.com
```

Comment:
Can you remove the class javadoc?

Reply:
stuff copied from quarkus-grpc where I didn't have adding headers disabled, will do